css/Z_INDEX.md

```diff
@@ -18,7 +18,9 @@ amp-user-notification                       |   1000      |   extensions/amp-use
 .-amp-image-lightbox-trans                  |   1001      |   extensions/amp-image-lightbox/0.1/amp-image-lightbox.css
 .-amp-sidebar-mask                          |   9998      |   extensions/amp-sidebar/0.1/amp-sidebar.css
 amp-sidebar                                 |   9999      |   extensions/amp-sidebar/0.1/amp-sidebar.css
-.-amp-lbv-mask                              |  2147483643 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
-.amp-lightboxed                             |  2147483644 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
+.-amp-lbv-mask                              |  2147483642 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
+.amp-lightboxed                             |  2147483643 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
+.-amp-lbv-click-area                        |  2147483644 |   extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css
```

Comment:
removed `click-area` layer.  I tried to expand `desc-box` layer to cover the full screen, and added eventListener to it, however it would prevent the click event pass to iframe.  My current solution is adding click event listener to `this.container_`, there's one drawback, for every other child of `this.container_` I need to call `event.stopPropagation()` in its event listener. Any thought on this?